src/backend/utils/adt/like_match.c

```diff
@@ -350,15 +350,26 @@ do_like_escape(text *pat, text *esc)
 
 	if (elen == 0)
 	{
-		/*
-		 * No escape character is wanted.  Double any backslashes in the
```

Comment:
Also, please follow the format used here for comments in your changes.

Reply:
Done.